### ISSUE TITLE
Duplicated data values when using mv command

### ISSUE THREAD

**Issue by unai-ttxu** [mlorenzo-stratio/vcli#5]
When I use `mv` command within vcli in order to move a certain secret, `data` key is duplicated. We can see the `mv` result in the following code:

```
{
  "request_id": "abb772b2-c5ac-4463-bad2-fb237cf9f371",
  "lease_id": "",
  "renewable": false,
  "lease_duration": 2764800,
  "data": {
    "data": {
      "mykeytab_keytab": "ThisIsMyKeytabValue",
      "myprincipal_principal": "myprincipal@TEST.COM"
    },
    "lease_duration": 2764800,
    "lease_id": "",
    "renewable": false,
    "request_id": "6e371564-33eb-6385-9808-3db7b5dd2304",
    "warnings": null
  },
  "wrap_info": null,
  "warnings": null,
  "auth": null
}
```

**Comment by ml0renz0** [comment]
I cannot reproduce this issue and now that I recall, may this be an old bug that was fixed? Are you running the last version of vcli?
I just added a new version that adds the possibility to check vcli's version:
```sh
$ vcli -h
Usage: /home/mlorenzo/bin/vcli -v <vault_host> -p <vault_port> -t <vault_token> [-i]
       /home/mlorenzo/bin/vcli --version
$ vcli --version
vcli v0.2.7
```

**Comment by ml0renz0** [comment]
I just fixed some bugs in *tree commands in the [0.2.9](https://github.com/mlorenzo-stratio/vcli/releases/tag/0.2.9) version. Try to test this issue against this latest version

**Comment by unai-ttxu** [comment]
Checked!

Thank you so much for the support!
